packages/react-components/react-menu/src/components/MenuItemLink/useMenuItemLink.ts

```diff
@@ -29,7 +41,7 @@ export const useMenuItemLink_unstable = (
       getIntrinsicElementProps('a', {
         ref,
         role: 'menuitem',
-        ...props,
+        ..._props,
```

Comment:
```suggestion
       // FIXME: casting because the root slot changes from div to a,
        // ideal solution would be to extract common logic from useMenuItem_unstable
        // and use it in both without assuming element type
        ...(baseState.root as ExtractSlotProps<Slot<'a'>>),
```